app/views/inthenewspoets/items/show.html.erb

```diff
@@ -94,11 +94,12 @@
   </div>
 <% end %>
 
-<% if @res["citation"].present? && @res["citation"]["works"].present?  %>
-  <h3>Related Works (<%= @res["citation"]["works"].count %>)</h3>
+<%# if @res["citation"].present? && @res["citation"]["title"].present?  %>
```

Comment:
Looks like this commented line can be dropped as the loop below iterates based on presence of the title key

Reply:
fixed